### DISCUSSION TITLE
ReactJS - Material de estudo, qual usar?

### DISCUSSION THREAD

**Discussion by victorsilent** [frontendbr/forum#580] [Perguntas e Respostas]
Olá pessoal :)
Eu estava procurando algum material para começar os estudos com React, dai vendo alguns cursos na udemy, alura e etc e acabei me deparando com o uso do react na versão v0.14.7 e o framework está na v15.5, dai dei uma lida no changelog para ver as alterações e me pareceram mudanças tranquilas, mas queria saber de vocês que ja usam ele, vale a pena pegar um material de estudo na versão v0.14.x? Pra ir pra v15 é tranquilo?

**Comment by expalmer** [comment]
Sim. vai firme

**Comment by willycamargo** [comment]
As mudanças não são tão drásticas, cara. Segue firme, isso não será um problema tão grande.

Recomendo instalar a mesma versão que o instrutor do curso for usar (isso vale para todas as dependencias), e quando for desenvolver algo novo, instale as versões mais atualizadas. No caso do react, qualquer erro de sintaxe os próprios warnings irão te avisar qual será a solução, e no caso de outros packages, hora ou outra algo pode quebrar, mas não acho que seria uma dificuldade que irá bloquear sua evolução.

**Comment by victorsilent** [comment]
Massa, eu continuei o curso, não tive problemas até agora :p
Vlw pessoal 😀

**Comment by leonardoelias** [comment]
@victorsilent tem um curso do @fdaciuk muito bom de React, da uma conversada com ele ;)

**Comment by ghost** [comment]
Eu comecei o [curso do Cod3r](https://www.cod3r.com.br/portal/courses/react-+-redux:-fundamentos-e-duas-apps-do-absoluto-zero-18) e curti muito, recomendo dar uma analisada.

**Comment by rictorres** [comment]
recomendo https://egghead.io/

**Comment by victorsilent** [comment]
Mais uma vez, vlwzão pelas dicas <3
Como material base de estudo usei o https://www.udemy.com/react-redux-pt/
Tava querendo ver mais sobre JS e o curso aborda a parte de back tambem, então está sendo uma mão na roda :p
Sobre o react em si, até agora ta de boa, mesmo preferindo 1000x o vue.js, mas como na empresa iremos usar o react, vai ter que ser isso mesmo kk

**Comment by matheusml** [comment]
Aproveitando a deixa, acabou de sair do forno:

[O Guia Completo do React e o seu Ecossistema](https://medium.com/tableless/o-guia-completo-do-react-e-o-seu-ecossistema-b31a10ecd84f)

**Comment by yanmagale** [comment]
Acabei descobrindo este final de semana este link material - [React Express](http://www.react.express/l)
